dataprocspawner/spawner.py

```diff
@@ -556,7 +559,7 @@ def _options_form_default(self):
     """
     base_html = get_base_cluster_html_form(
         self._list_gcs_files(self.dataproc_configs),
-        self.dataproc_locations_list.split(','),
+        self._validate_zones(self.region, self.dataproc_locations_list),
```

Comment:
Let's actually do this validation in the spawner __init__ function, so it only gets called once per user instead of on every spawn page load.

Reply:
Done